spec/node_spec.rb

```diff
@@ -142,18 +142,20 @@ module Ferrum
 
       context "when one of option with provided texts does not exist" do
         it "picks only existed options with no exception" do
-          expect(browser.at_xpath("//*[@id='form_title']").selected).to eq(["Mrs"])
-          expect(browser.at_xpath("//*[@id='form_title']").select(%w[Mrs SQL]).selected).to eq(["Mrs"])
+          expect(browser.at_xpath("//*[@id='form_title']").selected.map(&:text)).to eq(["Mrs"])
+          expect(browser.at_xpath("//*[@id='form_title']").select(%w[Mrs SQL]).selected.map(&:text)).to eq(["Mrs"])
         end
       end
 
       context "when select has multiple property" do
         it "picks options in select by match arguments as array" do
-          expect(browser.at_xpath("//*[@id='form_languages']").select(%w[SQL Ruby]).selected).to eq(%w[Ruby SQL])
+          expect(browser.at_xpath("//*[@id='form_languages']").select(%w[SQL Ruby]).selected.map(&:text))
+            .to eq(%w[Ruby SQL])
         end
 
-        it "picks options in select by match arguments as string" do
-          expect(browser.at_xpath("//*[@id='form_languages']").select("SQL, Ruby").selected).to eq(%w[Ruby SQL])
+        it "picks options in select by match arguments as strings" do
+          expect(browser.at_xpath("//*[@id='form_languages']").select("SQL", "Ruby").selected.map(&:text))
```

Comment:
from `select("SQL, Ruby")` to `.select("SQL", "Ruby")`
Let's use stringS for multiple selections, as same as here:
https://github.com/puppeteer/puppeteer/blob/caa9a1cafa6510acc2344cd1ae67fce4c5ee4df5/src/common/JSHandle.ts#L507